### PR TITLE
[OpenTelemetry.Exporter.Stackdriver] Release 1.0.0-beta.5 (#1579)

### LINKED ISSUE #1579: Inquiry on the new OpenTelemetry.Exporter.Stackdriver beta release
I'm writing to request a new beta release of `OpenTelemetry.Exporter.Stackdriver` that includes the changes from issue [#1414](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/issues/1414). These changes would significantly benefit our current project.

Could you consider rolling out a beta version with these updates? The current version [1.0.0-beta.4](https://www.nuget.org/packages/OpenTelemetry.Exporter.Stackdriver/1.0.0-beta.4) was released on 12/7/2022. Happy to assist in testing or any other way needed.

Thanks for your great work on this project!

### PR DESCRIPTION
**Pull request by StasPerekrestov**
Fixes #1579.

## Changes

Releases `OpenTelemetry.Exporter.Stackdriver` 1.0.0-beta.5

### CODE REVIEW

**Comment by vishweshbankwar** [comment]
@SergeyKanzhelev Could you please take a look as well?

**Review by vishweshbankwar — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+## 1.0.0-beta.5
+
+Released 2024-Feb-13
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md:7]:
```suggestion
Released 2024-Feb-15
```

Comment (vishweshbankwar) [reply src/OpenTelemetry.Exporter.Stackdriver/CHANGELOG.md:7]: @StasPerekrestov FYI - I am updating the date here

**Comment by vishweshbankwar** [comment]
@StasPerekrestov FYI - https://www.nuget.org/packages/OpenTelemetry.Exporter.Stackdriver/1.0.0-beta.5

**Comment by StasPerekrestov** [comment]
Thank you! Will test it tomorrow. It's a bit late over here.

**Comment by StasPerekrestov** [comment]
Everything looks much better now. Thank you for giving me a hand with this.

**Pull request merged**